pkg/utils/testutil/consts.go

```diff
@@ -16,7 +16,7 @@ package testutil
 
 const (
 	// APIAddress is the address of the API server used for testing.
-	APIAddress = "fake API address"
+	APIAddress = "1.0.0.1"
```

Comment:
Not referring to this explicitly, but the FakeHelmValues function (in testutil/kubernetes.go) is no longer used and could be dropped.